script.js

```diff
@@ -0,0 +1,76 @@
+var drawing;
+var content;
+var winCombos;
+var turn = 0;
+var theCanvas;
+var c;
+var cxt;
+var squaresFilled = 0;
+var y;
+
+window.onload=function(){
+  drawing = new Array();
+  content = new Array();
+  winCombos = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]];
+  for(var l = 0; l <= 8; l=l+1){
+  drawing[l] = false;
+  content[l]='';
+  }
```

Comment:
Formatting: please indent with for loop (within all curly braces actually).  It will really help (me and you).